docs/source/keywords.rst

```diff
@@ -378,6 +402,10 @@ Architecture Keywords
       * Optimize for SNB architecture
       * ``OFF``
 
+    * * ``Kokkos_ARCH_SPR``
+      * Optimize for Sapphire Rapids architecture
```

Comment:
Say Intel

Reply:
Nevermind I see we don't say anything for other Intel CPU architecture.  We can improve later.